harvestingkit/elsevier_package.py

```diff
@@ -638,6 +638,9 @@ def get_publication_date(self, xml_doc):
                     )
                     return start_date.strftime("%Y-%m-%d")
             import dateutil.parser
+            #dateutil.parser.parse cant process dates like April-June 2016
+            start_date = re.sub('([A-Z][a-z]+).*[A-Z][a-z]+ (\d{4})', 
```

Comment:
@fschwenn have you seen @tsgit's comment? I think fixed that it's good to merge.